mRemoteNG/Config/Putty/PuttySessionsRegistryProvider.cs

```diff
@@ -24,8 +24,6 @@ public class PuttySessionsRegistryProvider : AbstractPuttySessionsProvider
 
         public override string[] GetSessionNames(bool raw = false)
         {
-            if (PuttySessionsKey == null) return null;
-
             var sessionsKey = Registry.CurrentUser.OpenSubKey(PuttySessionsKey);
             if (sessionsKey == null) return new string[] { };
```

Comment:
Just a small perf. improvement: it could return `Array.Empty<string>` instead of  `new string[]{}`